SAGE Journals.js

```diff
@@ -83,15 +83,41 @@ function scrape(doc, url) {
 	var doi = ZU.xpathText(doc, '//meta[@name="dc.Identifier" and @scheme="doi"]/@content');
 	var filename = ZU.xpathText(doc, '//input[@name="downloadFileName"]/@value');
 	var post = "doi=" + encodeURIComponent(doi) + "&downloadFileName=" + filename + "&include=abs&format=ris&direct=false&submit=Download+Citation";
-	var pdfurl = "//" + doc.location.host + "/pdf/" + doi;
+	var pdfurl = "//" + doc.location.host + "/doi/pdf/" + doi;
 	//Z.debug(pdfurl);
 	//Z.debug(post);
 	ZU.doPost(risURL, post, function(text) {
+		//The publication date is saved in DA and the date first
+		//appeared online is in Y1. Thus, we want to prefer DA over T1
+		//and will therefore simply delete the later in cases both
+		//dates are present.
+		if (text.indexOf("DA  - ")>-1) {
+			text = text.replace(/Y1  - .*\r?\n/, '');
+		}
+		
 		var translator = Zotero.loadTranslator("import");
 		translator.setTranslator("32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7");
 		translator.setString(text);
 		translator.setHandler("itemDone", function(obj, item) {
+			//The subtitle will be neglected in RIS and is only present in
+			//the website itself. Moreover, there can be problems with
+			//encodings of apostrophs.
+			var subtitle = ZU.xpathText(doc, '//div[contains(@class, "publicationContentSubTitle")]/h1');
+			var title = ZU.xpathText(doc, '//div[contains(@class, "publicationContentTitle")]/h1');
+			if (title) {
+				item.title = title.trim();
+				if (subtitle) {
+					item.title += ': ' + subtitle.trim();
+				}
+			}
+			//The encoding of apostrophs in the RIS are incorrect and
+			//therefore we extract the abstract again from the website.
```

Comment:
right, titles are taken care of. I didn't make the connection. 
My concern was authors with apostrophs (d'Artagnan, say), but it appears those come through fine 
http://journals.sagepub.com/doi/abs/10.1177/003693300004500101
http://journals.sagepub.com/doi/abs/10.1177/0040571X0711000629
for example.